src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -26,40 +27,47 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
   roomUrl,
 }) => {
   const dispatch = useDispatch();
+
+  const schedulePrimaryClasses = classNames({
+    "ScheduleItem--primary": isCurrentEvent,
+  });
+
+  const scheduleItemTimeSectionClasses = classNames(
+    "ScheduleItem__time-section",
+    schedulePrimaryClasses
+  );
+
   return (
-    <div className="schedule-item-container">
-      <div className={`time-section ${isCurrentEvent ? "primary" : ""}`}>
-        <div>
-          <b>{formatUtcSeconds(event.start_utc_seconds)}</b>
-        </div>
-        <div>
+    <div className="ScheduleItem">
+      <div className={scheduleItemTimeSectionClasses}>
+        <span className="ScheduleItem__event-date">
+          {labelDayRoomSchedule(event.start_utc_seconds)}
+        </span>
+        <span className="ScheduleItem__event-time">
+          {formatUtcSeconds(event.start_utc_seconds)}
+        </span>
+        <span className="ScheduleItem__event-time">
           {formatUtcSeconds(
             event.start_utc_seconds + event.duration_minutes * 60
           )}
-        </div>
+        </span>
       </div>
-      <div className="event-section">
-        <div>
-          <div className={`${isCurrentEvent ? "primary" : ""}`}>
-            <div>
-              <b>{event.name}</b>
-            </div>
-            <div>
-              by <b>{event.host}</b>
-            </div>
-          </div>
-          <div className="event-description">
+      <div className="ScheduleItem__event-section">
+        <div className={schedulePrimaryClasses}>
+          <span className="ScheduleItem__event-name">{event.name}</span>
+          by <span className="ScheduleItem__event-host">{event.host}</span>
+          <span className="ScheduleItem__event-description">
             <RenderMarkdown text={event.description} />
-          </div>
+          </span>
```

Comment:
Why did we change this to use `span` rather than `div` here?

Reply:
The `span` after 'by' was necessary to prevent a line break. I've reverted the others. dc3f55a5c5b9c8dc6fd06fbf87da70238b8ac19a

Reply:
I asked a question about why we changed it, I didn't say to change it. PRs are a conversation, and questions are used to get the required context to make proper decisions. I didn't say reverting it was the best choice. I asked why it was done.

So asking again, why was it changed in the first place?